lupa/_lupa.pyx

```diff
@@ -156,6 +157,10 @@ def lua_type(obj):
         lua.lua_settop(L, old_top)
         unlock_runtime(lua_object._runtime)
 
+cdef inline int _len_as_int(Py_ssize_t obj) except -1:
+    if obj > <Py_ssize_t>LONG_MAX:
+        raise OverflowError
+    return <int>obj
```

Comment:
You could probably still get a C compiler warning here if `sizeof(long) == sizeof(Py_ssize_t)`, because it would detect that the condition is always false and, thus, useless.

BTW, why do you compare to `LONG_MAX` and not `INT_MAX` if you want to return an `int`?

Reply:
Is there a better way to deal with it or we should direct cast it?